### ISSUE TITLE
[FEATURE] Align text inside a label

### ISSUE THREAD

**Issue by fufexan** [elkowar/eww#597]
### Description of the requested feature

I would like to have the ability to set a custom alignment for wrapped text inside a label. Currently line breaks mean the text will continue from the beginning on the next line. I'd like it to, for example, be centered under the previous line.

I've tried playing with both `:xalign 0.5` and `:halign "center"` and none of them seem to be able to do this.

### Proposed configuration syntax

```clj
(label
  :text "Some very long text"
  :wrap true
  :textalign "center")
```

**Comment by Swexti** [comment]
+1 for this feature. Currently, I've been adding a label for each line and then center aligning them in the box.

**Comment by VuiMuich** [comment]
I believe there might be some gtk constraints at play, that eww has not a lot of possibilities to influence it.
Not entirely sure if this really is the case here, but I remember some issues with gtk exetending window height and giving eww no saying in it, when the font ascender and descender heights for the font size reserve more then the config sets for the window (i.e. in a task bar window).

**Comment by Swexti** [comment]
Yeah, that's a big issue in current widget handling in Eww. The boxes are expanded longer than they should when ANY element (not just text) exceeds the size of the box. This could definitely restrict the possibility of this feature.

**Comment by thfsilvab** [comment]
I see this issue is open but there's a "justify" property for the label that supposedly aligns it, though I've been trying to center the text with it, the label text remains anchored to the right. Does anyone get this same behavior?
